GVFS/GVFS.Installer.Mac/uninstall_vfsforgit.sh

```diff
@@ -82,6 +83,12 @@ function UnInstallVFSForGit()
         echo "$rmCmd..."
         eval $rmCmd || { echo "Error: Could not delete ${LIBRARYAPPSUPPORTDIRECTORY}. Delete it manually."; exit 1; }
     fi
+
+    if [ -d "${SERVICEAPPDIRECTORY}" ]; then
+        rmCmd="/bin/rm -Rf \"${SERVICEAPPDIRECTORY}\""
```

Comment:
Most of the other `rm` commands use `sudo` and this one doesn't, is that because this one is in the user's home directory?  Will this never need `sudo` then?

Reply:
> is that because this one is in the user's home directory?

Correct - these files are all user modifiable.

> Will this never need sudo then?

It shouldn't, unless something gets incorrect permissions... But maybe it is better to run this command as root in case something is set up unexpectedly, so we don't leave artifacts around for the next run.

Unless there are objections, I will update this to run as root.